src/oncall/ui/static/js/incalendar.js

```diff
@@ -1490,7 +1490,7 @@
             .append('<label class="label-col">Note: </label>')
             .append(
               $('<div class="input-col" />')
-              .append('<input type="text" id="inc-event-note" name="inc-event-note" value="' + (evt.note || '') + '" style="width:100%" /> ')
+              .append('<input type="text" id="inc-event-note" name="inc-event-note" value="' + (Handlebars.Utils.escapeExpression(evt.note) || '') + '" style="width:100%" /> ')
```

Comment:
We dont need to escape this. It's just putting text into the input box so it doesn't need to be escaped.